qiita_db/software.py

```diff
@@ -455,6 +482,38 @@ def add_publications(self, publications):
             qdb.sql_connection.TRN.add(sql, sql_params, many=True)
             qdb.sql_connection.TRN.execute()
 
+    @property
+    def environment_script(self):
+        """The script used to start the plugin environment
+
+        Returns
+        -------
+        str
+            The script used to start the environment
```

Comment:
Got it, would be useful to have that info!

On (Dec-04-15|13:29), Jose Navas wrote:
> > @@ -455,6 +482,38 @@ def add_publications(self, publications):
> >              qdb.sql_connection.TRN.add(sql, sql_params, many=True)
> >              qdb.sql_connection.TRN.execute()
> >  
> > +    @property
> > +    def environment_script(self):
> > +        """The script used to start the plugin environment
> > +
> > +        Returns
> > +        -------
> > +        str
> > +            The script used to start the environment
> 
> Path to the scrip - not necessarily full as it will be executed inside the environment in which the plugin lives, so it should be added in the path already.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1558/files#r46734528